regression/scripts/slipstream/Slipstream_Test.py

```diff
@@ -0,0 +1,295 @@
+    print('Cl difference')
+    print(diff_Cl)
+    assert  max(np.abs((sectional_lift_coeff - sectional_lift_coeff_true)/sectional_lift_coeff_true)) < 1e-6 
+    
+    # Plot pressure and lift distribution of wings 
+    plot_surface_pressure_contours(results,configs.base)
+    plot_lift_distribution(results,configs.base)
+    create_video_frames(results,configs.base, save_figure = False)
+    
+    return
+
+
+# ----------------------------------------------------------------------
+#   Analysis Setup
+# ----------------------------------------------------------------------
+
+def full_setup():
+
+    # vehicle data
+    vehicle  = vehicle_setup() 
+    configs  = configs_setup(vehicle)
+
+    # vehicle analyses
+    configs_analyses = analyses_setup(configs)
+
+    # mission analyses
+    mission  = mission_setup(configs_analyses,vehicle) 
+    missions_analyses = missions_setup(mission)
+
+    analyses = SUAVE.Analyses.Analysis.Container()
+    analyses.configs  = configs_analyses
+    analyses.missions = missions_analyses
+
+    return configs, analyses
+
+# ----------------------------------------------------------------------
+#   Define the Vehicle Analyses
+# ----------------------------------------------------------------------
+def analyses_setup(configs):
+
+    configs, analyses = full_setup()
+
+    simple_sizing(configs)
+
+    configs.finalize()
+    analyses.finalize()
+
+    # weight analysis
+    weights = analyses.configs.base.weights
+    breakdown = weights.evaluate()      
+
+    # mission analysis
+    mission = analyses.missions.base
+    results = mission.evaluate()
+
+    return analyses
+# ----------------------------------------------------------------------
+#   Define the Vehicle Analyses
+# ----------------------------------------------------------------------
+
+def analyses_setup(configs):
+
+    analyses = SUAVE.Analyses.Analysis.Container()
+
+    # build a base analysis for each config
+    for tag,config in configs.items():
+        analysis = base_analysis(config)
+        analyses[tag] = analysis
+
+    return analyses
+
+def base_analysis(vehicle):
+
+    # ------------------------------------------------------------------
+    #   Initialize the Analyses
+    # ------------------------------------------------------------------     
+    analyses = SUAVE.Analyses.Vehicle()
+
+    # ------------------------------------------------------------------
+    #  Basic Geometry Relations
+    sizing = SUAVE.Analyses.Sizing.Sizing()
+    sizing.features.vehicle = vehicle
+    analyses.append(sizing)
+
+    # ------------------------------------------------------------------
+    #  Weights
+    weights = SUAVE.Analyses.Weights.Weights_Tube_Wing()
+    weights.vehicle = vehicle
+    analyses.append(weights)
+
+    # ------------------------------------------------------------------
+    #  Aerodynamics Analysis
+    aerodynamics = SUAVE.Analyses.Aerodynamics.Fidelity_Zero()
```

Comment:
Yes, currently if there is a propeller, it will compute the effect depending on the placement of the propeller.  If no propeller type propulsor, no effect is modeled. I can toggle it.